src/main/java/com/avispl/symphony/dal/communicator/lg/lcd/LgLCDDevice.java

```diff
@@ -1,369 +1,402 @@
+
+		extendedStatistics.setControl(controllable);
+		extendedStatistics.setStatistics(statistics);
+		extendedStatistics.setDynamicStatistics(dynamicStatistics);
+
+		return Collections.singletonList(extendedStatistics);
+	}
+
+	/**
+	 * This method is recalled by Symphony to get the current monitor ID (Future purpose)
+	 *
+	 * @return int This returns the current monitor ID.
+	 */
+	public int getMonitorID() {
+		return monitorID;
+	}
+
+	/**
+	 * This method is is used by Symphony to set the monitor ID (FUture purpose)
+	 *
+	 * @param monitorID This is the monitor ID to be set
+	 */
+	public void setMonitorID(int monitorID) {
+		this.monitorID = monitorID;
+	}
+
+	/**
+	 * This method is used to get the current display power status
+	 *
+	 * @return powerStatus This returns the calculated xor checksum.
+	 */
+	private LgLCDConstants.powerStatusNames getPower() {
+
+		try {
+			byte[] response = send(LgLCDUtils.buildSendString((byte) monitorID, LgLCDConstants.commands.get(LgLCDConstants.commandNames.POWER), LgLCDConstants.commands.get(commandNames.GET)));
+
+			LgLCDConstants.powerStatusNames power = (LgLCDConstants.powerStatusNames) digestResponse(response, LgLCDConstants.commandNames.POWER);
+
+			if (power == null) {
+				return LgLCDConstants.powerStatusNames.OFF;
+			} else {
+				return power;
+			}
+		} catch (Exception e) {
+			if (this.logger.isErrorEnabled()) {
+				this.logger.error("error during get power send", e);
+			}
+		}
+		return null;
+	}
+
+	protected void powerON() {
+		try {
+			byte[] response = send(
+					LgLCDUtils.buildSendString((byte) monitorID, LgLCDConstants.commands.get(LgLCDConstants.commandNames.POWER), LgLCDConstants.powerStatus.get(LgLCDConstants.powerStatusNames.ON)));
+
+			digestResponse(response, LgLCDConstants.commandNames.POWER);
+		} catch (Exception e) {
+			if (this.logger.isDebugEnabled()) {
+				this.logger.debug("error during power OFF send", e);
+			}
+		}
+	}
+
+	protected void powerOFF() {
+		try {
+			byte[] response = send(
+					LgLCDUtils.buildSendString((byte) monitorID, LgLCDConstants.commands.get(LgLCDConstants.commandNames.POWER), LgLCDConstants.powerStatus.get(LgLCDConstants.powerStatusNames.OFF)));
+
+			digestResponse(response, LgLCDConstants.commandNames.POWER);
+		} catch (Exception e) {
+			if (this.logger.isDebugEnabled()) {
+				this.logger.debug("error during power ON send", e);
+			}
+		}
+	}
+
+	/**
+	 * This method is used to get the current display input
+	 *
+	 * @return inputNames This returns the current input.
+	 */
+	private LgLCDConstants.inputNames getInput() {
+		try {
+			byte[] response = send(LgLCDUtils.buildSendString((byte) monitorID, LgLCDConstants.commands.get(commandNames.INPUT), LgLCDConstants.commands.get(commandNames.GET)));
+
+			LgLCDConstants.inputNames input = (LgLCDConstants.inputNames) digestResponse(response, LgLCDConstants.commandNames.INPUT);
+
+			return input;
+		} catch (Exception e) {
+			System.out.println("Connect exception");
```

Comment:
Please replace System.out.println with logger. Check the other places as well

Reply:
I fixed it